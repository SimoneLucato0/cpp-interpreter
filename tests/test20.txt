FOR fun false or FUN return super AND else if class CLASS IF NIL var OR ELSE RETURN while true nil PRINT for WHILE TRUE VAR this print and THIS FALSE SUPER