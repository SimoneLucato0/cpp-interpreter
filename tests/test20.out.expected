IDENTIFIER FOR null
FUN fun null
FALSE false null
OR or null
IDENTIFIER FUN null
RETURN return null
SUPER super null
IDENTIFIER AND null
ELSE else null
IF if null
CLASS class null
IDENTIFIER CLASS null
IDENTIFIER IF null
IDENTIFIER NIL null
VAR var null
IDENTIFIER OR null
IDENTIFIER ELSE null
IDENTIFIER RETURN null
WHILE while null
TRUE true null
NIL nil null
IDENTIFIER PRINT null
FOR for null
IDENTIFIER WHILE null
IDENTIFIER TRUE null
IDENTIFIER VAR null
THIS this null
PRINT print null
AND and null
IDENTIFIER THIS null
IDENTIFIER FALSE null
IDENTIFIER SUPER null
EOF  null
